tests/v1/engine/test_async_llm.py

```diff
@@ -13,6 +13,7 @@
                 allow_module_level=True)
 
 ENGINE_ARGS = AsyncEngineArgs(model="meta-llama/Llama-3.2-1B",
+                              enforce_eager=True,
```

Comment:
This shouldn't be necessary, right?

Reply:
Its not needed, but it makes the tests go faster. Takes like 45s to startup with torch.compile